Source/JavaScriptCore/assembler/AssemblerCommon.h

```diff
@@ -28,6 +28,9 @@
 #include "OSCheck.h"
 #include <optional>
 #include <wtf/Atomics.h>
+#if CPU(ARM_THUMB2)
+#include <wtf/MathExtras.h>
```

Comment:
I'm not sure why, but without this change, the build fails on armv7 with an error saying that `WTF::roundUpToMultipleOf` cannot be found.
Since this file actually uses `WTF::roundUpToMultipleOf`, I think including this header file makes sense.